test/Interop/C/swiftify-import/sized-by.swift

```diff
@@ -0,0 +1,52 @@
+// REQUIRES: swift_feature_SafeInteropWrappers
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -disable-availability-checking -plugin-path %swift-plugin-dir -o %t/CountedBy.swiftmodule -I %S/Inputs -enable-experimental-feature SafeInteropWrappers %s
+
+// Check that ClangImporter correctly infers and expands @_SwiftifyImport macros for functions with __sized_by parameters.
```

Comment:
I wonder if dumping the swift signatures with something like ide tests would be a more direct way to test this than calling the functions. Here, I wonder if some implicit conversions can weaken some of these test cases.

Reply:
That did make the test a bit more straightforward. I still kept the frontend invocation as well though, since swift-ide-test doesn't check the syntax or types of the macro expansion.